packages/app-layout/src/vaadin-app-layout.d.ts

```diff
@@ -182,6 +182,14 @@ declare class AppLayout extends ElementMixin(ThemableMixin(ControllerMixin(HTMLE
    */
   closeDrawerOn: string;
 
+  /**
+   * Set to true to disable closing the drawer on Escape press
+   * Pressing Escape only closes the drawer if it is opened as overlay
```

Comment:
Consider adding a clarification like we have in "Navigation" section e.g.

```suggestion
   * Pressing Escape only closes the drawer if it is opened as overlay on small devices.
```